internal/sysfs/select_windows.go

```diff
@@ -14,6 +16,8 @@ const wasiFdStdin = 0
 // pollInterval is the interval between each calls to peekNamedPipe in pollNamedPipe
 const pollInterval = 100 * time.Millisecond
 
+var kernel32 = syscall.NewLazyDLL("kernel32.dll")
```

Comment:
no big deal (better for clarity), but this may mean that `loadlibrary()` and the proc lookup may be invoked more than once for `kernel32.dll` (but on a low-level it may be cached anyway?)

Reply:
not sure, but I like your train of thought!